docs/lesson/settings.rst

```diff
@@ -0,0 +1,120 @@
+
+2. Fill out the fields under **Basic Settings**.
+
+.. figure:: /images/basic_settings.png
+   :alt: Basic Settings
+   :scale: 30 %
+
+   *Fig. 1*
+
+3. To see a preview of your exploration and how it would appear to a learner as they search in the library, click on the **Preview Summary** button. The **Title** and **Goal** fields are displayed in the preview. 
+
+.. figure:: /images/preview_summary.png
+   :alt: Preview Summary Card
+   :scale: 30 %
+
+   *Fig. 2*
+
+.. _advancedfeatures:
+
+Advanced Features
+------------------
+
+**Automatic Text-to-speech**
+
+This setting is available under **Advanced Features**. The Automatic Text-to-speech feature is enabled by default and automatically generates audio from the text in your cards.
+
+When the feature is enabled, learners will see an **Audio** button as they play through the exploration, which will generate a computerized reading of your card.
+If you disable the feature, the button is not visible to learners.
+
+.. figure:: /images/text_speech.png
+   :alt: Audio button
+   :scale: 40 %
+
+   *Fig. 3*
+
+.. tip::
+
+   Disable the text-to-speech feature if your exploration content consists of multiple languages.
+
+**Correctness Feedback**
+
+The **Correctness Feedback** feature lets students know when they have selected a right answer by displaying a short animation that looks like this:
+
+.. figure:: /images/correctness_feedback.png
+   :alt: Correctness feedback
+   :scale: 40 %
+
+   *Fig. 4*
+
+
+1. To enable this feature, click on |settings|, go to **Advanced Features** and click the button next to **Correctness Feedback** so that it is 'On'.
+
+2. In the Editing |pencil| screen, you will now see an option to identify an answer group as correct in the **Learner's Answers and Oppia's Responses** section. Checkmark the box within the correct answer group. You can add several possible answers to a group if there is more than one correct answer.
+
+.. |pencil| image:: /images/pencil_icon.png
+            :scale: 25 % 
+
+.. figure:: /images/correct_answer_group.png
+   :alt: Correct answer group
+   :scale: 40 %
+
+   *Fig. 5*
+
+3. Save your changes. Now the learner will be able to view the 'Correct' animation if they input an answer you have identified as 'correct'.
+
+.. _emailpref:
+
+Changing Email Preferences
+----------------------------
+
+The default setting in Oppia is for an exploration creator to receive notifications whenever a learner submits :ref:`feedback <feedback>` or :ref:`suggestions <suggestions>` for your exploration.
+
+You can stop these notifications by clicking the **Mute** button under **Feedback/Suggestion Email Preferences**.
+
+.. figure:: /images/email_preferences.png
+   :alt: Mute email preferences
+   :scale: 40 %
+
+   *Fig. 6*
+
+.. _deletingexploration:
+
+Delete an Exploration
+----------------------
+
+Finally, you can delete your exploration if you wish by clicking the |deleteexploration| button at the bottom of the **Settings** page.
```

Comment:
Should we mention that they cannot delete this exploration once it's published?

Reply:
I remember Sean mentioning that this feature might change in the future - so not to specifically say that, just to mention where the 'Delete Exploration' button was.